core/trino-spi/src/main/java/io/trino/spi/block/Block.java

```diff
@@ -311,7 +311,9 @@ default Block getLoadedBlock()
     }
 
     /**
-     * Gets the direct child blocks of this block.
```

Comment:
@dain should review this

Reply:
backed out

Reply:
@djsstarburst can you please create a new PR with this change?